src/WebJobs.Extensions.DurableTask/Options/AzureStorageOptions.cs

```diff
@@ -144,15 +145,31 @@ public void ValidateHubName(string hubName)
 
         private static string GetTaskHubErrorString(string hubName)
         {
-            return $"Task hub name '{hubName}' should contain only alphanumeric characters and have length between {MinTaskHubNameSize} and {MaxTaskHubNameSize}.";
+            return $"Task hub name '{hubName}' should contain only alphanumeric characters, start with a letter, and have length between {MinTaskHubNameSize} and {MaxTaskHubNameSize}.";
         }
 
         internal bool IsSanitizedHubName(string hubName, out string sanitizedHubName)
```

Comment:
These changes would also need to be ported to the Web App scale controller, as well as Functions V2 runtime (at least until the scale controller fix roles out).